app/locales/rtl-languages.js

```diff
@@ -0,0 +1,19 @@
+// This file contains all RTL languages ISO code just in case someone added one of this languages
+// Used in OnBoarding1.js file to set app to rtl if user pick one of these languages
+
+const rtlLanguages = [
```

Comment:
this should be detectable from `i18n.dir('ar')`

Reply:
Removed and reading the direction directly from i18next